smartmontools/drivedb.h

```diff
@@ -4557,6 +4557,13 @@ const drive_settings builtin_knowndrives[] = {
     "", "",
     "-v 9,msec24hour32 -v 188,raw16 -v 240,msec24hour32"
   },
+  { "Seagate Enterprise Capacity 3.5 HDD", // untested
```

Comment:
@OddMagnet, Tha name of harddisk is `Seagate Enterprise Capacity 3.5 HDD v7 SED` according to produck manual. https://github.com/smartmontools/smartmontools/files/14470974/Seagate_Enterprise_Capacity_3.5_HDD_v7_Self-Encryption_512E_model_ST12000NM0127.pdf